CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
+# 0.6.0
+
+* Enhancement - Renamed default version and branch to `mainline`.
```

Comment:
```suggestion
```

The default branch changed to `main`, but I don't see this as something worth calling out in the changelog.

Reply:
Okay yeah, fair enough.